src/main/java/net/atos/zac/flowable/FlowableTaskService.java

```diff
@@ -37,7 +37,7 @@
 
 @ApplicationScoped
 @Transactional
-public class TakenService {
+public class FlowableTaskService {
```

Comment:
Don't want to be picky, but this service is already in the context of a flowable package, so it seems like repeating the package name in the class name. Is that what we want? Should it maybe not be renamed in the context of where this class is used - if that makes it clearer?

Reply:
I think this is better. It is a service that is a wrapper around Flowable's TaskService. I renamed it because I want to create a new `TaskService` class which is what you would expect from a task service. :-)

Reply:
More or less the same discussion as with zaken vs cases. A task in ZAC has different meanings depending on the context: a ZAC task is a task in the context of a zaak. A Flowable task is a much more technical Flowable-specific thing. So this is a service for 'Flowable tasks'. Hence the name `FlowableTaskService`. :-)